addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -1091,6 +1091,10 @@ var SnippetEditor = Widget.extend({
                     this.dragState.columnWidth = parseFloat(this.$target[0].scrollWidth);
                     this.dragState.columnHeight = parseFloat(this.$target[0].scrollHeight);
                 }
+                // Taking the column borders into account.
+                const style = window.getComputedStyle(this.$target[0]);
+                this.dragState.columnWidth += parseFloat(style.borderLeft) + parseFloat(style.borderRight);
+                this.dragState.columnHeight += parseFloat(style.borderTop) + parseFloat(style.borderBottom);
```

Comment:
Why not adding the borders directly at the place where the `width/height` is added in `this.dragState` (in the `if/else`).
Also same comment as above, could you use `offsetHeight` / `offsetWidth` ?

Reply:
It was just to avoid repeating the computation and to have a smaller diff than 
```js
// If the column comes from a snippet that doesn't toggle the
// grid mode on drag, store its width and height to use them
// when the column goes over a grid dropzone.
const style = window.getComputedStyle(this.$target[0]);
const borderX =  parseFloat(style.borderLeft) + parseFloat(style.borderRight);
const borderY = parseFloat(style.borderTop) + parseFloat(style.borderBottom);
const isImageColumn = gridUtils._checkIfImageColumn(this.$target[0]);
if (isImageColumn) {
    // Store the image width and height if the column only
    // contains an image.
    const imageEl = this.$target[0].querySelector('img');
    this.dragState.columnWidth = parseFloat(imageEl.scrollWidth) + borderX;
    this.dragState.columnHeight = parseFloat(imageEl.scrollHeight) + borderY;
} else {
    this.dragState.columnWidth = parseFloat(this.$target[0].scrollWidth) + borderX;
    this.dragState.columnHeight = parseFloat(this.$target[0].scrollHeight) + borderY;
}
```
I prefer doing what is common to the if/else outside of them :slightly_smiling_face: